google/cloud/bigquery/big_query_read_client.gcpcxx.pb.h

```diff
@@ -0,0 +1,204 @@
+   */
+  StreamRange<::google::cloud::bigquery::storage::v1::ReadRowsResponse>
+  ReadRows(std::string const& read_stream, std::int64_t offset);
+
+  /**
+   * Creates a new read session. A read session divides the contents of a
+   * BigQuery table into one or more streams, which can then be used to read
+   * data from the table. The read session also specifies properties of the
+   * data to be read, such as a list of columns or a push-down filter describing
+   * the rows to be returned.
+   *
+   * A particular row can be read by at most one stream. When the caller has
+   * reached the end of each stream in the session, then all the data in the
+   * table has been read.
+   *
+   * Data is assigned to each stream such that roughly the same number of
+   * rows can be read from each stream. Because the server-side unit for
+   * assigning data is collections of rows, the API does not guarantee that
+   * each stream will return the same number or rows. Additionally, the
+   * limits are enforced based on the number of pre-filtered rows, so some
+   * filters can lead to lopsided assignments.
+   *
+   * Read sessions automatically expire 24 hours after they are created and do
+   * not require manual clean-up by the caller.
+   *
+   * @param request
+   * [::google::cloud::bigquery::storage::v1::CreateReadSessionRequest](https://github.com/googleapis/googleapis/blob/dff6e4625d4ea0a16fc44d3b9be115219c403f07/google/cloud/bigquery/storage/v1/storage.proto#L110)
+   * @return
+   * [::google::cloud::bigquery::storage::v1::ReadSession](https://github.com/googleapis/googleapis/blob/dff6e4625d4ea0a16fc44d3b9be115219c403f07/google/cloud/bigquery/storage/v1/stream.proto#L47)
+   */
+  StatusOr<::google::cloud::bigquery::storage::v1::ReadSession>
+  CreateReadSession(
+      ::google::cloud::bigquery::storage::v1::CreateReadSessionRequest const&
+          request);
+
+  /**
+   * Reads rows from the stream in the format prescribed by the ReadSession.
+   * Each response contains one or more table rows, up to a maximum of 100 MiB
+   * per response; read requests which attempt to read individual rows larger
+   * than 100 MiB will fail.
+   *
+   * Each request also returns a set of stream statistics reflecting the current
+   * state of the stream.
+   *
+   * @param request
+   * [::google::cloud::bigquery::storage::v1::ReadRowsRequest](https://github.com/googleapis/googleapis/blob/dff6e4625d4ea0a16fc44d3b9be115219c403f07/google/cloud/bigquery/storage/v1/storage.proto#L135)
+   * @return
+   * [::google::cloud::bigquery::storage::v1::ReadRowsResponse](https://github.com/googleapis/googleapis/blob/dff6e4625d4ea0a16fc44d3b9be115219c403f07/google/cloud/bigquery/storage/v1/storage.proto#L185)
+   */
+  StreamRange<::google::cloud::bigquery::storage::v1::ReadRowsResponse>
+  ReadRows(::google::cloud::bigquery::storage::v1::ReadRowsRequest request);
```

Comment:
I think we should group overloads if possible, so can this function be moved up to be near the other `ReadRows()` overload?